### PR TITLE
updates to views, acceptance tests for manage_course, create sections and those unit tests, and assign user and those unit tests

### PR DESCRIPTION
**Pull request by ekruse09**

### CODE REVIEW

File: supercreative/create_sections/section.py [hunk 1]
```diff
@@ -1,33 +1,36 @@
-from supercreative.models import Course, Section
+from supercreative.models import Course, Section, User, UserCourseAssignment
 from django.core.exceptions import ObjectDoesNotExist
 
 
-def create_section(section_id, course, section_type):
+def create_section(section_id, course, section_type, user):
```
Comment (N7Ironclad) [inline supercreative/create_sections/section.py]: I don't think we should require a user parameter here. It should be possible to create a section and later assign a user to it. Perhaps make it an optional parameter.

File: supercreative/create_sections/section.py [hunk 2]
```diff
@@ -38,5 +41,14 @@ def create_section(section_id, course, section_type):
     )
     new_section.save()
 
+    new_user_course_assignment = UserCourseAssignment(
```
Comment (N7Ironclad) [inline supercreative/create_sections/section.py]: Either remove creating a UserCourseAssignment here if we remove the user parameter or if the user parameter becomes optional, make creating the UserCourseAssignment only happen if it's non-null.

File: supercreative/tests/Acceptance_Tests/test_manage_course.py
Comment (N7Ironclad) [inline supercreative/tests/Acceptance_Tests/test_manage_course.py:1]: Will require some changes to reflect the changes made to create_sections parameters.

File: supercreative/tests/Unit_Tests/test_create_section.py
Comment (N7Ironclad) [inline supercreative/tests/Unit_Tests/test_create_section.py:1]: Will require some changes to reflect the changes made to create_sections parameters.

**Review by N7Ironclad — CHANGES_REQUESTED** [review]

File: supercreative/create_sections/section.py (hunk 1 above)
Comment (ekruse09) [reply supercreative/create_sections/section.py]: me and brennan talked about making it a required parameter, we can discuss it more at the meeting

**Pull request closed without merging**